src/Waives.Http/HttpRequestMessageBuilder.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net.Http;
+
+namespace Waives.Http
+{
+    internal static class HttpRequestMessageBuilder
```

Comment:
💪 I think this class duplicates code from the tests, `ReliableRequestSenderFacts.ARequestWithContentAndHeader()` (and maybe elsewhere?). It would be great to remove the duplication and have everything use the same thing :+1:

Reply:
They do similar but annoyingly not quite the same job. `ReliableRequestSenderFacts.ARequestWithContentAndHeader()` returns an `HttpRequestMessageTemplate`. The builder returns an `HttpRequestMessage`. I think removing the duplication would actually involve making it more complex.